app/views/partials/_item_form.html.erb

```diff
@@ -0,0 +1,19 @@
+
+<%= form_tag path, method: method do %>
+  <%= label_tag :name %>
+  <%= text_field_tag :name, name %>
+
+  <%= label_tag :description %>
+  <%= text_field_tag :description, description %>
+
+  <%= label_tag :price %>
+  <%= text_field_tag :price, price %>
+
+  <%= label_tag :image %>
+  <%= text_field_tag :image, image %>
+
+  <%= label_tag :inventory %>
+  <%= text_field_tag :inventory, inventory %>

```

Comment:
Another option is to use a number_field_tag instead of a text_field_tag. It comes with some built-in flash messages. We could add it to (#83) as a refactor possibility when we have completed all user stories.